tensorboard/plugins/debugger_v2/debugger_v2_plugin_test.py

```diff
@@ -89,6 +96,14 @@ def setUp(self):
         self.plugin = debugger_v2_plugin.DebuggerV2Plugin(context)
         wsgi_app = application.TensorBoardWSGI([self.plugin])
         self.server = werkzeug_test.Client(wsgi_app, wrappers.BaseResponse)
+        # The multiplexer reads data asynchrnously on a separate thread, so
+        # as not to block the main thread of the TensorBoard backend. During
+        # unit test, we disable the asynchronous behavior, so that we can
+        # load the debugger data synchronously on the main thread, which leads
+        # to determinisic behavior.
+        tf.compat.v1.test.mock.patch.object(
+            debug_data_multiplexer, "run_in_background", lambda target: target()
```

Comment:
This mocked-out `run_in_background` has pretty different semantics from
the real one: it should probably return `None` and not propagate errors
to the caller (either printing an error or failing the test would be
fine with me). Otherwise, the code under test could accidentally depend
on synchronous execution.